engine/render/src/render/render_script.cpp

```diff
@@ -2722,9 +2722,13 @@ namespace dmRender
 
         REGISTER_BUFFER_CONSTANT(COLOR0_BIT,  COLOR_BIT); // For backwards compatability
         REGISTER_BUFFER_CONSTANT(COLOR0_BIT,  COLOR0_BIT);
-        REGISTER_BUFFER_CONSTANT(COLOR1_BIT,  COLOR1_BIT);
-        REGISTER_BUFFER_CONSTANT(COLOR2_BIT,  COLOR2_BIT);
-        REGISTER_BUFFER_CONSTANT(COLOR3_BIT,  COLOR3_BIT);
+        // These depend on driver and context support
+        if (dmGraphics::IsMultiTargetRenderingSupported(graphics_context))
+        {
+            REGISTER_BUFFER_CONSTANT(COLOR1_BIT,  COLOR1_BIT);
+            REGISTER_BUFFER_CONSTANT(COLOR2_BIT,  COLOR2_BIT);
+            REGISTER_BUFFER_CONSTANT(COLOR3_BIT,  COLOR3_BIT);
+        }
```

Comment:
This behaviour is similar to the floating point formats etc. If we have support for multiple outputs we add the enums